comp/aggregator/demultiplexer/demultiplexerimpl/demultiplexer.go

```diff
@@ -21,8 +28,10 @@ import (
 	"github.com/DataDog/datadog-agent/comp/serializer/compression"
 	"github.com/DataDog/datadog-agent/pkg/aggregator"
 	"github.com/DataDog/datadog-agent/pkg/aggregator/sender"
+	"github.com/DataDog/datadog-agent/pkg/config"
```

Comment:
Les ensure we do not use the `pkg/config` and use the config component. If we do not have access to it we can add it to the dependencies struct 😄

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/25367/commits/145321e38f4609ce60aef40c8370267f9b5072b8